tensorhive/config.py

```diff
@@ -86,14 +86,19 @@ class ServicesConfig():
     because instances below are depending on it
     '''
     from tensorhive.core.services.MonitoringService import MonitoringService
+    from tensorhive.core.services.ProtectionService import ProtectionService
     from tensorhive.core.monitors.Monitor import Monitor
     from tensorhive.core.monitors.GPUMonitoringBehaviour import GPUMonitoringBehaviour
+    from tensorhive.core.violation_handlers.ProtectionHandler import ProtectionHandler
+    from tensorhive.core.violation_handlers.MessageSendingBehaviour import MessageSendingBehaviour
     ENABLED_SERVICES = [
         MonitoringService(monitors=[
             Monitor(GPUMonitoringBehaviour())
             # Add more monitors here
 
-        ], interval=1.0)
+        ], interval=1.0),
+        ProtectionService(handler=ProtectionHandler(injected_behaviour=MessageSendingBehaviour()),
```

Comment:
Maybe the argument could be called behaviour instead of injected_behaviour. BTW, what if we want to define multiple handlers?